### PR TITLE
57 implement user input of contact info

### LINKED ISSUE #57: Implement user input of contact info
- Implement contact info input on search result page, which Safetec can later use to contact the user
- Implement sending of email to Safetec with contact info
- Ask Safetec if they want to store the contact info in the database
  - If so, implement `ContactInfo` model for storing it
  - Find out what sort of consent we need to ask of the user to do this

### PR DESCRIPTION
**Pull request by KatjaSi**

### CODE REVIEW

File: base/templates/base/export_control_law.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {% block breadcrumbs %}
 <nav aria-label="breadcrumb">
   <ol class="breadcrumb">
-    <li class="breadcrumb-item"><a href="{% url 'presentation' %}">Home</a></li>
+    <li class="breadcrumb-item"><a href="{% url 'export_control_law' %}">Home</a></li>
```
Comment (danielmursa) [inline base/templates/base/export_control_law.html]: Why did you change was correct before?

File: contacts/migrations/0003_alter_contact_phone.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1.1 on 2022-10-14 18:08
```
Comment (danielmursa) [inline contacts/migrations/0003_alter_contact_phone.py]: Try deleting all migration files and leaving your DB, and restart migrations again, this way you will have only one migration and not more

File: contacts/models.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+from django.forms import CharField
+from datetime import datetime
+from phonenumber_field.modelfields import PhoneNumberField
+
+class Contact(models.Model):
+    name = models.CharField(max_length=128)
+    email = models.CharField(max_length=64)
+    phone = PhoneNumberField(null=True, blank = True)
+    message = models.TextField(blank=True)
+    date_created = models.DateTimeField(default=datetime.now)
```
Comment (danielmursa) [inline contacts/models.py]: date_created: types.DateTimeField = models.DateTimeField(_("Date created"), auto_now_add=True, db_index=True)
    date_updated: types.DateTimeField = models.DateTimeField(_("Date updated"), auto_now=True, db_index=True)

File: contacts/models.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+from email.mime import message
+from xmlrpc.client import DateTime
+import black
+from django.db import models
+from django.forms import CharField
+from datetime import datetime
+from phonenumber_field.modelfields import PhoneNumberField
+
+class Contact(models.Model):
+    name = models.CharField(max_length=128)
```
Comment (danielmursa) [inline contacts/models.py]: use the same format in regulation model

File: contacts/urls.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    path('thank-you/', TemplateView.as_view(template_name="thank_you.html"), name = 'thank_you')
+]
+
+if settings.DEBUG:
+    import debug_toolbar  # type: ignore
+
+    # Allow error pages to be tested
+    urlpatterns += [
+        path("__debug__/", include(debug_toolbar.urls)),
+    ]
```
Comment (danielmursa) [inline contacts/urls.py]: you can delete this part because it is arleady exist in main URL file

File: contacts/views.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+def send_contact(request):
+    contact_form = ContactForm()
+    if request.POST:
+        contact_form = ContactForm(request.POST)
+        if contact_form.is_valid():
+            contact_form.save()
+            return redirect(reverse('search_page'))
+    context = {"form": contact_form}
+    return render(request, "contact_form.html", context) 
+
```
Comment (danielmursa) [inline contacts/views.py]: this part should be in the Class and not outside

File: contacts/models.py (hunk 3 above)
Comment (KatjaSi) [reply contacts/models.py:10]: error

I get this error. it works in regulations, so why do i get this?

**Review by danielmursa — APPROVED** [review]
